{"name": "操作日志列表-失败", "status": "passed", "attachments": [{"name": "log", "source": "9cf10cdf-ba91-4c80-b1ea-06418f74ea1f-attachment.txt", "type": "text/plain"}], "start": 1690436285729, "stop": 1690436285773, "uuid": "f2ee4970-cb85-44e9-8a37-8ae696ab8700", "historyId": "ad1e9b3def66782ad162eeb06d1cf45f", "testCaseId": "6185372399dbf1b63eb947da37d9a5ce", "fullName": "opsPC.test_logManage#test_pc_log_list_002", "labels": [{"name": "story", "value": "日志管理"}, {"name": "feature", "value": "运维管理平台"}, {"name": "parentSuite", "value": "opsPC"}, {"name": "suite", "value": "test_logManage"}, {"name": "host", "value": "cale"}, {"name": "thread", "value": "10244-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "opsPC.test_logManage"}]}